src/website/views/components/forms/text-input.pug

```diff
@@ -34,11 +46,19 @@ p.-text
 h4 Left Aligned
 .example.-mb--3
   .-p--3
-    .a-inputWrapper.-icon--left(style="max-width:20rem")
-      input(type="text", class="a-input", placeholder="Placeholder", value="Sample Text")
-      .a-icon.-text--muted
-        svg
-          use(xlink:href='#icon-edit')
+    .-d--flex.-flex--row.-align-items--end
+      .a-inputWrapper.-icon--left.-mr--3(style="max-width:20rem")
+        input(type="text", class="a-input", placeholder="Placeholder", value="Sample Text")
+        .a-icon.-text--muted
+          svg
+            use(xlink:href='#icon-edit')
+      .m-formGroup.-labeled(style="max-width:20rem")
```

Comment:
Remove all the label examples from the Icon section. No need to call it out here

Reply:
Done, but i put these examples because they have different markup